src/js/check-access.js

```diff
@@ -1,6 +1,66 @@
 import { checkValidity } from './validate';
+import { getApiToken } from './auth';
 
+const ENDPOINT = 'https://fac-dev.app.cloud.gov/sac/accessandsubmission';
 const FORM = document.forms[0];
+let addedContactNum = 1; // Counter for added contacts
+
+function submitForm() {
+  const formData = serializeFormData(new FormData(FORM));
+
+  let preparedData = {};
+  preparedData['certifying_auditee_contact'] =
+    formData.auditee_certifying_official_email;
+  preparedData['certifying_auditor_contact'] =
+    formData.auditor_certifying_official_email;
+  preparedData['auditee_contacts'] = contactsToArray(
+    formData,
+    'auditee_contacts_email'
+  );
+  preparedData['auditor_contacts'] = contactsToArray(
+    formData,
+    'auditor_contacts_email'
+  );
+
+  const headers = new Headers();
+  headers.append('Content-type', 'application/json');
+
+  getApiToken().then((token) => {
```

Comment:
Not a blocker, but there's a [`queryApi`](https://github.com/GSA-TTS/FAC-Frontend/blob/2dbf04b1776bb002ad2a8f054a85b5a05c173665/src/js/api.js#L3) method that does this, so there's no need to keep writing this for every API call.

Reply:
Doh! Let me make that update.